src/types.d.ts

```diff
@@ -105,4 +105,19 @@ type BundleMDXOptions = {
    * ```
    */
   globals?: Record<string, string | ModuleInfo>
+  /**
+   * The current working directory for the mdx bundle.
+   *
+   * All relative paths are resolved from this directory by esbuild. The supplied
+   * folder doesn't have to exist, useful if your MDX content doesn't live on the
+   * disk.
```

Comment:
Ah, if this is the use case you might mention that we default to a folder that doesn't exist, so the only real time someone would want to provide a value is when the `cwd` actually *does* exist, because we have a default and it wouldn't make much sense for someone to provide a different directory that doesn't exist 😅

Reply:
Yeah maybe my wording here is wrong.

Supplying a `cwd` _activates_ resolution outside the `files` object, so yse you would probabbly always set it to something that does exist otherwise everything will be imported from `../` at a minimum.